cookbook/security/form_login_setup.rst

```diff
@@ -458,7 +462,8 @@ firewall matches *all* URLs, including ``/login_check``). If ``/login_check``
 doesn't match any firewall, you'll receive a ``Unable to find the controller
 for path "/login_check"`` exception.
 
-**4. Multiple firewalls don't share security context**
+4. Multiple Firewalls Don't share Security Context
```

Comment:
this and (5) seem to not be related to form login, but just general security pitfalls.